src/com/company/Main.java

```diff
@@ -12,14 +12,14 @@ public static void main(String[] args){
 
         String txtfile = args[0];
```

Comment:
txtfile should be renamed in camel case. Also wouldn't it be better if we rename the txtfile to something like txtFilePath or so?

Reply:
yeah!!
done with that..